pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationState.java

```diff
@@ -37,6 +39,10 @@
      */
     String getAuthRole() throws AuthenticationException;
 
+    default List<String> getAuthRoles() throws AuthenticationException {
+        return Collections.singletonList(getAuthRole());
```

Comment:
What if the getAuthRole returns null or empty string I think this should return an empty list